### ISSUE TITLE
Babel does not compile files in platformAssets/runtime

### ISSUE THREAD

**Issue by TheDuc** [pavjacko/renative#522]
Files in platformAssets are not compiled by babel. This poses a problem on older platforms with `fontManager.web.js` that contains an arrow function (or any other runtime files).
This can locally be fixed by adding a virtual plugin
```
"babel_platformassets_fix": {
    "webpack": {
        "modulePaths": [
            "platformAssets/runtime"
        ]
    }
},
```
to plugins in renative.json. This adds the `platformAssets/runtime` folder to the include rules of babel.
But I believe this should be fixed in renative itself

**Comment by pavjacko** [comment]
Thanks @TheDuc fixed in this commit:
https://github.com/pavjacko/renative/commit/be795306fcb3fa66f73828ac197cc428fc15a802

will be shipped with `0.29.1`

**Comment by TheDuc** [comment]
thanks for the quick fix as usual!
